### DISCUSSION TITLE
Please add GithubActions section to app

### DISCUSSION THREAD

**Discussion by big-toni** [community/community#21064] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.73.0 (72950773)
Device: iPhone 13 mini (iOS 15.5)
</details>

**Comment by iskae** [comment]
Hi @big-toni 👋 

Thanks for reaching out and for your feedback ❤️ 

GitHub Actions is definitely one of the most requested features that we receive 😊 Good news though, we are working hard to bring GitHub Actions to mobile, so please stay tuned for our future releases 🎉 

Cheers!
